src/App.js

```diff
@@ -14,6 +15,7 @@ const router = createBrowserRouter([
     children: [
       { index: true, element: <HomePage /> },
       { path: 'signin', element: <SignInPage /> },
+      { path: 'settings', element: <SettingsPage /> },
```

Comment:
I put a /setting path in the browser url input and the page works. Could you protect the whole route to make it not possible to bypass auth validation?